selfdrive/boardd/boardd.cc

```diff
@@ -57,6 +57,8 @@ bool loopback_can = false;
 cereal::HealthData::HwType hw_type = cereal::HealthData::HwType::UNKNOWN;
 bool is_pigeon = false;
 const uint32_t NO_IGNITION_CNT_MAX = 2 * 60 * 60 * 24 * 3;  // turn off charge after 3 days
+const uint32_t VBATT_START_CHARGING = 12000;
+const uint32_t VBATT_PAUSE_CHARGING = 10500;
```

Comment:
This is still really low, the battery would already be dead by the time EON stops charging. I would recommend 11800 or thereabout.

Absent a temporal/debounce component, yes EON will stop charging during brief periods of high ignition-off load, but it will resume when Vbatt recovers.

Reply:
11000-+500 is what is measures so it only trips the low limit